client/state/ui/selectors.js

```diff
@@ -103,12 +103,25 @@ export function isSectionLoading( state ) {
  * Returns true if the current section is isomorphic.
  *
  * @param  {Object}  state Global state tree
- * @return {bool}    True if current section is isomorphic
+ * @return {Boolean}    True if current section is isomorphic
  *
  * @see client/sections
  */
 export function isSectionIsomorphic( state ) {
-	return get( getSection( state ), 'isomorphic', false );
+	return !! get( getSection( state ), 'isomorphic', false );
```

Comment:
Do we need the `!!`? The third argument makes sure we're falling back to returning `false` if no `isomorphic` field is found.